src/arduino/arduino.ino

```diff
@@ -0,0 +1,79 @@
+#include <DS1307RTC.h>
+#include <TimeLib.h>
+#include <Time.h>
+#include "components/Pressure.cpp"
+
+// pins
+#define COMPRESSOR_PIN 9
+#define VALVE_1_PIN 8
+#define SENSOR_PIN A0         // Analog
+#define COMP_SWITCH_PIN 11    // button #1
+#define VALVE_1_SWITCH_PIN 10 // button #2
+
+// constants
+#define VOLTAGE_CONST 5
+#define P_MAX 145 // psi
+#define P_MIN 0
+
+#define SYSTEM_PRESSURE 82    // average pressure we want to maintain
```

Comment:
What are the units here? PSI? Cause below this is set to 10 but its implied that we want to have it be 3 psi. Might wanna clariffy documentation here